src/styles/blocks/display.scss

```diff
@@ -0,0 +1,46 @@
+.display {
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__watch {
+    height: 80vh;
+    width: 80vh;
+    border: 5px dashed #000;
+    border-radius: 50%;
+    position: relative;
+
+    &-center::after {
+      content: "";
+      border: 20px solid orange;
+      border-radius: 100%;
+      position: absolute;
+      top: calc(50% - 20px);
+      left: calc(50% - 20px);
+      z-index: 1;
+    }
+
+    &-minutes {
+      border: 10px solid blue;
+      width: calc(50% / 2);
+      border-radius: 5px;
+      position: absolute;
+      top: calc(50% - 10px);
+      left: 50%;
+      animation: rotate 60s linear infinite;
+      transform-origin: left center;
+    }
+
+    &-seconds {
+      border: 10px solid green;
+      height: calc(50% - 20px);
+      border-radius: 5px;
+      position: absolute;
+      left: calc(50% - 10px);
+      animation: rotate 5s linear infinite;
+      transform-origin: bottom;
```

Comment:
would be nice to create a common class for same rules here

Reply:
common class it's like:
```html
  <div class="display">
      <div class="display__watch">
        <div class="display__watch-center"></div>
        <div class="display__watch-minutes common"></div>
        <div class="display__watch-seconds common"></div>
      </div>
    </div>
```
no need to create another wrappers